lib/v2/douyin/user.js

```diff
@@ -0,0 +1,50 @@
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    // 获取 uid
+    const { uid } = ctx.params;
+    const url = 'https://www.douyin.com/user/' + uid;
+
+    // 访问 url
+    const browser = await require('@/utils/puppeteer')();
+    const page = await browser.newPage();
+    await page.goto(url);
+    const html = await page.evaluate(() => document.documentElement.innerHTML);
+    browser.close();
+
+    // 解析 html
+    const $ = cheerio.load(html);
```

Comment:
```suggestion
    // 访问 url
    const response = await ctx.cache.tryGet(
        url,
        async () => {
            const browser = await require('@/utils/puppeteer')();
            const page = await browser.newPage();
            await page.goto(url);
            const html = await page.evaluate(() => document.documentElement.innerHTML);
            browser.close();
            return html;
        },
        config.cache.routeExpire,
        false
    );

    // 解析 html
    const $ = cheerio.load(response);
```
Please add cache to puppeteer's response